cpp/src/arrow/util/bitmap.h

```diff
@@ -453,7 +459,8 @@ class ARROW_EXPORT Bitmap : public util::ToStringOstreamable<Bitmap>,
     return bitmaps[0].length();
   }
 
-  std::shared_ptr<Buffer> buffer_;
```

Comment:
In general, we should watch out for classes that take copies of `shared_ptr<T>` without a good reason to; this was a good example